mount.go

```diff
@@ -32,6 +33,30 @@ import (
 	"github.com/jacobsa/timeutil"
 )
 
+func isMetadataPrefetchOnMountEnabled(metadataPrefetchMode string) bool {
+	switch metadataPrefetchMode {
+	case config.MetadataPrefetchOnMountSynchronous, config.MetadataPrefetchOnMountAsynchronous:
+		return true
+	}
+	return false
+}
+
+func isTypeCacheEnabled(metadataCacheTTL time.Duration, typeCacheMaxSizeMB int) bool {
+	if metadataCacheTTL <= 0 || typeCacheMaxSizeMB == 0 || typeCacheMaxSizeMB < -1 {
+		return false
+	}
+
+	return true
+}
+
+func isStatCacheEnabled(metadataCacheTTL time.Duration, statCacheMaxSizeMB uint64) bool {
```

Comment:
should we add these helper methods and their test cases somewhere apt inside `internal` directory ? trying to keep main.go clean.